updated doc on changes to localtest (#7060)
Altinn/altinn-studio: LOCALAPP.md

The app and local platform services are now running locally. The app can be accessed on <http://altinn3local.no>.

Log in with a test user, using your app name and org name. This will redirect you to the app.

### Configuration of test data

In some cases your application might differ from the default setup and require custom changes to the test data available. 

> This section contains the most common changes required by the app developers in localtest.

Reviewer: ```suggestion
This section contains the most common changes.
```

> #### Adjust authentication level of logged in test user

Reviewer: ```suggestion
#### Adjust authentication level of logged in test user
This would be required if your app requires a higher than default authentication level. You can also use this to give the user a lower authentication level if you want to test the app behaviour for those.
```

> 1. Open the `src/development/LocalTest/Controllers/HomeController.cs` in your preffered tool. 

Reviewer: ```suggestion
1. Open the `src/development/LocalTest/Controllers/HomeController.cs` in your preffered text editor or IDE. 
```

2. Find the function `LogInTestUser` 
3. Modify this line `claims.Add(new Claim(AltinnCoreClaimTypes.AuthenticationLevel, "2", ClaimValueTypes.Integer32, issuer));`,
by exchanging `"2"` for a string containing your required authentication level.
4. Save and close the file
5. Restart LocalTest

> #### Add a missing role for a test user

Reviewer: ```suggestion
#### Add a missing role for a test user
This would be required if your app requires a role which none of the test users have.
```
